src/main/resources/config/liquibase/changelog/20250626094500_changelog.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
+
+    <changeSet id="20250626094500-1" author="Alphar">
+        <createTable tableName="exercise_version">
+            <!-- Common fields -->
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="author_id" type="bigint">
+                <constraints nullable="false" />
+            </column>
+            <column name="exercise_id" type="bigint">
+                <constraints nullable="false" />
+            </column>
+            <column name="content" type="longtext">
```

Comment:
```suggestion
            <column name="content" type="json">
```
Both MySQL and PostgreSQL should have a JSON datatype. Can we use this here instead?

Reply:
Example for a JSON-column in Artemis: https://github.com/ls1intum/Artemis/blob/27f327a06f09f28b956f9161d1fa51ca7d96cb11/src/main/java/de/tum/cit/aet/artemis/lecture/domain/LectureTranscription.java#L28

Reply:
This PR also contains an example: https://github.com/ls1intum/Artemis/pull/11055#discussion_r2175194532